src/utils/train.py

```diff
@@ -139,7 +140,15 @@ def train(
                     ocr_flag=ocr_flag,
                 )
             if batch_num % config.save_every == 0:
-                torch.save(model.state_dict(), os.path.join(validation_wandb_path, f"batch_{batch_num}.ckpt"))
+                torch.save(
+                    {
+                        "scheduler": scheduler.state_dict(),
+                        "model": model.state_dict(),
+                        "optimizer": optimizer.state_dict(),
+                        "loss": loss,
+                    },
```

Comment:
That's very useful, please also add loading these states on the model checkpoint load

Reply:
what model checkpoint load do you mean?